src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/util/FilesystemUtil.java

```diff
@@ -52,5 +52,12 @@ public static void setScriptPermission(InstanceConfiguration config, String scri
                 .addArgument("755")
                 .addArgument(String.format("bin/%s", scriptName));
         ProcessUtil.executeScript(config, command);
+
+        command = new CommandLine("sed")
+                .addArguments("-i''")
+                .addArgument("-e")
+                .addArgument("1s:.*:#!/usr/bin/env bash:", false)
+                .addArgument(String.format("bin/%s", scriptName));
+        ProcessUtil.executeScript(config, command);
```

Comment:
Interesting... did you have problems with the location of the bash executable?

Reply:
Yeah, I run NixOS which doesn't have a `/bin/bash`, POSIX only requires the presence of `/bin/sh` and the best practice is to use `/usr/bin/env` to find any other specific shell, from what I've been able to grok.

Obviously this is not otherwise related to the PR, but is required for me and other NixOS users to run the plugin.